crates/trigger/src/cli.rs

```diff
@@ -54,6 +55,14 @@ pub struct FactorsTriggerCommand<T: Trigger<B::Factors>, B: RuntimeFactorsBuilde
     )]
     pub log: Option<PathBuf>,
 
+    /// Whether to refresh logs when restarted.
+    #[clap(
+        name = REFRESH_LOGS,
```

Comment:
Maybe consider a `SPIN_` prefix for this?  Going by the original issue, some folks will be treating this as a user preference and setting it in `.profile` rather than on the `spin up` command line, so it would be good to have it make sense out of the Spin context perhaps?

(sorry forgot to think of this while reviewing)